src/lib/common/homestore_config.fbs

```diff
@@ -220,14 +220,17 @@ table Consensus {
     // Minimum log gap a replica has to be from leader before joining the replica set.
     min_log_gap_to_join: int32 = 30;
     
-    // amount of time in seconds to wait on data write before fetch data from remote;
-    wait_data_write_timer_sec: uint32 =  30 (hotswap);
+    // amount of time in millis to wait on data write before fetch data from remote;
+    wait_data_write_timer_ms: uint64 = 1500 (hotswap);
     
-    // Leadership expiry 120 seconds
-    leadership_expiry_ms: uint32 = 120000;
+    // Leadership expiry (=0 indicates 20 times heartbeat period), set -1 to never expire
+    leadership_expiry_ms: int32 = 0;
```

Comment:
it is a bit tricky as heartbeat period is not expose.
Suggesting lets expose 
`heart_beat_interval_` in HS configuration then `election_timeout_upper_bound_`/`election_timeout_lower_bound_` and `leadership_expiry_ms` configuration as multiple of `heart_beat_interval_`

Reply:
Not sure I got your comment, we expose heartbeat_period_ms as HS config and also elect_timeout_low and high. So leadership_expiry by default is 20 times heartbeat_period unless someone explicitly sets an absolute value.